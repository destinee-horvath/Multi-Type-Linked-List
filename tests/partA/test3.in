NEW 2
hello
world
NEW 3
apples
3
5.4
NEW 4
10
9.5
b
a
VIEW ALL
REMOVE 0
REMOVE 1
REMOVE 2
REMOVE 0