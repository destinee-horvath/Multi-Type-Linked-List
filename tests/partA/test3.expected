List 0: hello -> world
List 1: apples -> 3 -> 5.40
List 2: 10 -> 9.50 -> b -> a
Number of lists: 3
List 0
List 1
List 2
List 0 has been removed.

Number of lists: 2
List 1
List 2
List 1 has been removed.

Number of lists: 1
List 2
List 2 has been removed.

Number of lists: 0
INVALID COMMAND: REMOVE
